spawns.lua

```diff
@@ -12,6 +12,7 @@ spawns = {
     ["Africa, Morocco, Marrakech"] = { x = 1920, y =  640 },
     ["Africa, Senegal, Dakar"] = { x = 1860, y = 810 },
     ["Africa, South Africa, Johannesburg"] = { x = 2335, y = 1260 },
+    ["Asia, Korea, Seoul"] = { x = 3440, y = 560 },
```

Comment:
Please keep this alphabetically ordered, so this should come after `["Asia, India, Delhi"]`